README.md

```diff
@@ -257,71 +254,22 @@ my-cluster-zookeeper-nodes    ClusterIP   None             <none>        2181/TC
 
 The service you're interested in is `my-cluster-kafka-bootstrap:9092`. We will use it to deploy the Data Index Service later.
 
-#### Create Required Topics
+#### Kafka Topics
 
-Having the cluster up and running, the next step is to create the `Kafka Topic`s required by the Data Index Service: 
+If Strimzi Operator is installed in the namespace, Kogito Operator will create the `Kafka Topic`s required by the Data Index Service: 
 
 - `kogito-processinstances-events`
 - `kogito-usertaskinstances-events` 
 - `kogito-processdomain-events`
 - `kogito-usertaskdomain-events`
 
-To create the required topics, in the OpenShift Web Console go to the Installed Operators, Strimzi Operator, Kafka Topic tab. 
-
-From there, create a new `Kafka Topic` and name it as `kogito-processinstances-events` like in the example below:
-
-```yaml
-apiVersion: kafka.strimzi.io/v1beta1
-kind: KafkaTopic
-metadata:
-  name: kogito-processinstances-events
-  labels:
-    strimzi.io/cluster: my-cluster
-  namespace: kogito
-spec:
-  partitions: 10
-  replicas: 3
-  config:
-    retention.ms: 604800000
-    segment.bytes: 1073741824
-```
-And then do the same for the others topics listed above.
-
-To check if everything was created successfully run the following command:
-
-```bash
-$ oc describe kafkatopic/kogito-processinstances-events
-
-Name:         kogito-processinstances-events
-Namespace:    kogito
-Labels:       strimzi.io/cluster=my-cluster
-Annotations:  <none>
-API Version:  kafka.strimzi.io/v1beta1
-Kind:         KafkaTopic
-Metadata:
-  Creation Timestamp:  2019-08-28T18:09:41Z
-  Generation:          2
-  Resource Version:    5673235
-  Self Link:           /apis/kafka.strimzi.io/v1beta1/namespaces/kogito/kafkatopics/kogito-processinstances-events
-  UID:                 0194989e-c9bf-11e9-8160-0615e4bfa428
-Spec:
-  Config:
-    message.format.version:  2.3-IV1
-    retention.ms:            604800000
-    segment.bytes:           1073741824
-  Partitions:                10
-  Replicas:                  1
-  Topic Name:                kogito-processinstances-events
-Events:                      <none>
-```
-
 Now that you have the required infrastructure, it's safe to deploy the Kogito Data Index Service.
 
 ### Install Data Index Service
 
 #### Retrieve Kafka internal url
 
-Having configured the Kafka Operator, you'll need the Kafka service URI to install the Data Index Service. 
+Having configured the Kafka Operator, you'll need the Kafka service URI or custom resource name to install the Data Index Service. 
```

Comment:
I would put 'Kafka Custom Resource name' to be more explicit

Reply:
Sure. Thanks.